lib/sass/calcite-web/components/_sliders.scss

```diff
@@ -0,0 +1,185 @@
+  @include range-style;
+  &:hover {
+    // webkit
+    &::-webkit-slider-runnable-track { background-color: $track-hover-bgcolor; }
+    &::-webkit-slider-thumb { @include range-hover-thumb-style; }
+    // ff
+    &::-moz-range-track { background-color: $track-hover-bgcolor; }
+    &::-moz-range-thumb { @include range-hover-thumb-style; }
+    // ie
+    &::-ms-fill-upper { background-color: $track-hover-bgcolor }
+    &::-ms-fill-lower { background-color: $track-hover-bgcolor }
+    &::-ms-thumb { @include range-hover-thumb-style; }
+  }
+  &:focus, &:active {
+    // webkit
+    &::-webkit-slider-runnable-track { background-color: $track-hover-bgcolor; }
+    &::-webkit-slider-thumb { @include range-focus-thumb-style; }
+    &::-webkit-slider-thumb:hover { @include range-focus-thumb-hover-style; }
+    // ff
+    &::-moz-range-track { background-color: $track-hover-bgcolor; }
+    &::-moz-range-thumb { @include range-focus-thumb-style; }
+    &::-moz-range-thumb:hover { @include range-focus-thumb-hover-style; }
+    // ie
+    &::-ms-fill-upper { background-color: $track-hover-bgcolor }
+    &::-ms-fill-lower { background-color: $track-hover-bgcolor }
+    &::-ms-thumb { @include range-focus-thumb-style; }
+    &::-ms-thumb:hover { @include range-focus-thumb-hover-style; }
+  }
+}
+
+// webkit styles
+input[type="range"]::-webkit-slider-runnable-track { @include track-style; }
+input[type="range"]::-webkit-slider-thumb {
+  @include thumb-style;
+  &:hover { @include thumb-hover-style; }
+  &:focus, &:active {
+    @include thumb-focus-style;
+    &:hover { @include thumb-focus-hover-style; }
+  }
+}
+// make tap target larger on touch devices
+@supports (-webkit-overflow-scrolling: touch) {
+  input[type="range"]::-webkit-slider-thumb {
+    height:22px;
+    width:22px;
+    margin-top:-10px;
+  }
+}
+// ff styles
+input[type="range"]::-moz-range-track { @include track-style; }
+input[type="range"]::-moz-range-thumb {
+  @include thumb-style;
+  height: 12px;
+  width: 12px;
+  &:hover { @include thumb-hover-style; }
+  &:focus, &:active {
+    @include thumb-focus-style;
+    &:hover { @include thumb-focus-hover-style; }
+  }
+}
+
+// ie styles
+input[type="range"]::-ms-fill-upper { background-color: $track-bgcolor; }
+input[type="range"]::-ms-fill-lower { background-color: $track-bgcolor; }
+input[type="range"]::-ms-track {
+  @include track-style;
+  height:4px;
+  border:0px solid transparent;
+  color: transparent;
+
+}
+input[type="range"]::-ms-thumb {
+  @include thumb-style;
+  height: 12px;
+  width: 12px;
+  margin-top:-2px;
+  &:hover { @include thumb-hover-style; }
+  &:focus, &:active {
+    @include thumb-focus-style;
+    &:hover { @include thumb-focus-hover-style; }
+  }
+}
+
+// don't include progress for now
+// input[type="range"]::-moz-range-progress { background-color: $track-progress-bgcolor; }
+// input[type="range"]::-ms-fill-lower { background-color: $track-progress-bgcolor; }
+
+@if $include-sliders == true {
```

Comment:
We should move all of the actual selectors (lines 94 - 182)  into here so that if users set `$include-sliders: false`, they don't get any generated CSS

Reply:
💫